packages/react-utils/src/useWindowSize.test.ts

```diff
@@ -0,0 +1,51 @@
+import { act, renderHook } from '@testing-library/react'
+import { beforeEach, expect, test } from 'vitest'
+import { useWindowSize } from './useWindowSize'
+
+beforeEach(() => {
+  window.innerWidth = 800
+  window.innerHeight = 600
+  window.dispatchEvent(new Event('resize'))
+})
+
+const dispatch = (width: number, height: number) => {
+  window.innerWidth = width
+  window.innerHeight = height
+  window.dispatchEvent(new Event('resize'))
+}
```

Comment:
suggestion: Could shorten by defining `dispatch` first and then using within `beforeEach`:

```ts
const dispatch = (width: number, height: number) => {
  window.innerWidth = width
  window.innerHeight = height
  window.dispatchEvent(new Event('resize'))
}

beforeEach(() => {
  dispatch(800, 600)
})
```